README.md

```diff
@@ -1,6 +1,12 @@
 # State of Our Networks 2019
 
-This repository has organizing materials for the upcoming State of Our Networks, for any questions email orga@ournetworks.ca!
+This repository has organizing materials for the Our Networks **RE: Infrastructures** on **September 20-22, 2019**, for any questions email orga@ournetworks.ca!
+
+**September 19** - Load-in // Workshopping
+**September 20-22** - Conference, [Toronto Media Arts Centre](https://www.tomediaarts.org/), 32 Lisgar Street  
+**September 20-24** - Exhibition
```

Comment:
Is it clear that Exhibition is same venue?

Reply:
I think it is fine...? Also we're not representing it like this on the website, which by and large is where folks look